src/Resources/assets/admin/js/index.js

```diff
@@ -0,0 +1,8 @@
+import 'semantic-ui-css/components/dropdown';
```

Comment:
```patch
--- src/Resources/assets/admin/js/index.js
+++ src/Resources/private/admin/js/app.js
```

I'm not sure if it's part of new bundle notation, that allows this `Resource/assets` path, but it should be either `Resources/private` or `/assets` in the root if plugin's symfony version allows it

Reply:
AFAIR it doesn't change anything as the paths are configured in the `webpack.config.js`. We can go with `Resource/private` (as we do in Sylius) and change it to the `/assets` once the plugin structure updated.